invite/views.py

```diff
@@ -35,10 +35,14 @@ def reset(request):
                 password=form.cleaned_data['password'],
             )
             if user is not None:
-                login(request, user)
-                # Redirect to main edit dashboard
-                return HttpResponseRedirect(
-                    app_settings.INVITE_SIGNUP_SUCCESS_URL)
+                form = forms.LoginForm()
+                return render(
+                    request,
+                    'invite/login.html',
+                    {
+                        'login_form': form,
+                    }
+                )
```

Comment:
I think rather than turn this page _into_ the login screen, we probably need to redirect, as it was doing before, but just change the destination to the login screen. Otherwise, won't this same view receive the login credentials and not actually log the user in?

Reply:
So basically I'm thinking this whole section would just change to:
`return HttpResponseRedirect(reverse('invite:login'))`

Reply:
Hmm..I think you are right. Since I just have to redirect the user to login page, I can use HttpResponseRedirect instead of render.